xbmc/guilib/GUIEditControl.cpp

```diff
@@ -747,3 +747,11 @@ void CGUIEditControl::SetFocus(bool focus)
   g_Windowing.EnableTextInput(focus);
   CGUIControl::SetFocus(focus);
 }
+
+std::string CGUIEditControl::GetDescriptionByIndex(int index) const
+{
+  if (index == 0)
+    return GetDescription();
+
+  return GetLabel2();
```

Comment:
we should return Label2 only if index is 1 otherwise return an empty string.
```
if (index == 0)
  return GetDescription()
else if(index == 1)
  return GetLabel2()

return "";
```